app1/admin.py

```diff
@@ -1,3 +1,19 @@
-from django.contrib import admin
+#from django.contrib import admin
 
 # Register your models here.
+from django.contrib import admin
+from .models import YourModel  # Import your model(s) here
+
+@admin.register(YourModel)
+class YourModelAdmin(admin.ModelAdmin):
+    list_display = ('field1', 'field2', 'field3')  # Add fields you want to display in the admin list view
+    search_fields = ('field1', 'field2')  # Add fields you want to search in the admin
```

Comment:
**Suggestion**: Enhance the 'search_fields' attribute by including additional fields for improved search functionality. <br> **Code Suggestion**: 
 ```
 +    search_fields = ('field1', 'field2')
 ```